### ISSUE TITLE
Bottom Bar Error Msg

### ISSUE THREAD

**Issue by AGoh2019** [microsoft/vscode#230035]
Type: <b>Bug</b>

A text  message "Reactivating terminals" keeps running.

VS Code version: Code 1.93.1 (38c31bc77e0dd6ae88a4e9cc93428cc27a56ba40, 2024-09-11T17:20:05.685Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz (8 x 2419)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.77GB (6.80GB free)|
|Process Argv|--crash-reporter-id 12067a16-8102-43e5-8034-4ad53ba49405|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (59)</summary>

Extension|Author (truncated)|Version
---|---|---
rust-bundle|1Yi|1.0.0
fsharp-interactive-datascience|and|1.0.12
vscode-django|bat|1.15.0
c-cpp-compile-run|dan|1.0.60
vscode-eslint|dba|3.0.10
python-environment-manager|don|1.2.4
python-extension-pack|don|1.7.0
rust-syntax|dus|0.6.1
vscode-html-css|ecm|2.0.10
dependi|fil|0.7.10
code-runner|for|0.12.2
c-cpp-runner|fra|9.4.9
fsharp-language-server|geo|0.1.28
vscode-javac|geo|0.2.46
Ionide-fsharp|Ion|7.21.2
vscode-csharp-snippets|jor|1.1.0
vsc-python-indent|Kev|1.18.0
csharpextensions|kre|1.7.3
csdevkit|ms-|1.10.18
csharp|ms-|2.45.25
vscode-dotnet-runtime|ms-|2.1.6
vscodeintellicode-csharp|ms-|2.1.11
autopep8|ms-|2024.0.0
black-formatter|ms-|2024.2.0
debugpy|ms-|2024.10.0
pylint|ms-|2023.10.1
python|ms-|2024.14.1
vscode-pylance|ms-|2024.9.2
jupyter|ms-|2024.8.1
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.19
vscode-jupyter-cell-tags|ms-|0.1.9
vscode-jupyter-slideshow|ms-|0.1.6
cmake-tools|ms-|1.19.52
cpptools|ms-|1.21.6
cpptools-extension-pack|ms-|1.3.0
vscode-typescript-next|ms-|5.7.20240927
autodocstring|njp|0.6.1
java|red|1.35.0
LiveServer|rit|5.7.9
rust-analyzer|rus|0.3.2121
rust-pack|Swe|0.3.38
tabnine-vscode|Tab|3.163.0
even-better-toml|tam|0.19.2
python-extended-snippets|tus|0.0.1
cmake|twx|0.0.17
vscode-lldb|vad|1.10.0
intellicode-api-usage-examples|Vis|0.2.8
vscodeintellicode|Vis|1.3.1
vscode-gradle|vsc|3.16.4
vscode-java-debug|vsc|0.58.0
vscode-java-dependency|vsc|0.24.0
vscode-java-pack|vsc|0.29.0
vscode-java-test|vsc|0.42.0
vscode-maven|vsc|0.44.0
vscode-spring-initializr|vsc|0.11.2
jinja|who|0.0.8
JavaScriptSnippets|xab|1.8.0
pretty-ts-errors|Yoa|0.6.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementsc:30995553
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
724cj586:31013169
a69g1124:31058053
dvdeprecation:31068756
dwnewjupytercf:31046870
newcmakeconfigv2:31071590
impr_priority:31102340
nativerepl2:31139839
refactort:31108082
pythonrstrctxt:31112756
flighttreat:31134774
wkspc-onlycs-t:31132770
nativeloc1:31134641
wkspc-ranged-t:31125599
cf971741:31144450
pme_test_t:31118333
defaultse:31146405
iacca2:31144504
5fd0e150:31146321

```

</details>

**Comment by gjsjohnmurray** [comment]
Please use the Start Extension Bisect command to investigate whether the problem is being caused by one of your extensions.

**Comment by AGoh2019** [comment]
Pls show me how to do it, step by step.
--- Andrew Goh
 

    On Sunday, 29 September 2024 at 12:05:53 am SGT, John Murray ***@***.***> wrote:  
 
 


Please use the Start Extension Bisect command to investigate whether the problem is being caused by one of your extensions.

**Comment by gjsjohnmurray** [comment]
Like for all VS Code commands, open Command Palette (on Windows the F1 key will do this, or Ctrl+Shift+P), then start typing the command. The list will reduce as you type. When you see the one you want, pick it.

The follow the on-screen instructions.

**Comment by karthiknadig** [comment]
Duplicate of: https://github.com/microsoft/vscode-python/issues/23922
see https://github.com/microsoft/vscode-python/issues/23922#issuecomment-2293911979 for options to unblock.
